.github/workflows/pull-request-ci.yaml

```diff
@@ -0,0 +1,319 @@
+
+    # Cache catapult's common tools
+    - name: cache.catapult-common-tools
+      uses: actions/cache@v2
+      with:
+        path: ${{ github.workspace }}/catapult/modules/common/bin
+        key: ${{ runner.os }}-catapult-common-tools
+
+    # Deploy k8s cluster
+    - run: make k8s
+      working-directory: catapult
+      env:
+        GKE_CLUSTER_NAME: kubecf-ci-${{ github.run_id }}-${{ matrix.backend }}
+        OWNER: ${{ github.repository_owner }}
+        DEBUG_MODE: false
+        GKE_NODE_COUNT: 1
+        GKE_PREEMPTIBLE: true
+        GKE_INSTANCE_TYPE: n1-highcpu-16
+
+    # Providing kubeconfig for debugging
+    - name: export KUBECONFIG
+      run: |
+        set -o errexit -o nounset -o pipefail
+        cd "build${BACKEND}"
+        source .envrc
+        echo "::set-env name=KUBECONFIG::${KUBECONFIG}"
+        # Debugging aid; should be safe as the cluster will be torn down
+        # at the end of the run.
+        cat "${KUBECONFIG}" | gzip | base64 --wrap=0
+      working-directory: catapult
+
+    - name: Add tools to path
+      run: |
+        echo "::add-path::${TOOLS_DIR}"
+
+    # TBD: Don't generate values.yaml
+    - name: Generate KubeCF Configuration
+      run: |
+        set -o errexit -o nounset -o pipefail
+
+        make -C ${{ github.workspace }}/catapult kubecf-gen-config
+
+        cd ${{ github.workspace }}/catapult/build${BACKEND}
+
+        gomplate --context .=scf-config-values.yaml <<"EOF" \
+          > ${{ github.workspace }}/kubecf/dev/kubecf/kubecf-values.yaml
+        {{- /* Disable brain minibroker tests */}}
+        {{- define "minibroker-tests" }}
+        properties:
+          brain-tests:
+            acceptance-tests-brain:
+              tests:
+                minibroker:
+                  {{- range slice "mariadb" "mongodb" "postgres" "redis" }}
+                  {{ . }}:
+                    enabled: false
+                  {{- end }}
+        {{- end }}
+        {{- $ = tmpl.Exec "minibroker-tests" | yaml | merge $ }}
+
+        {{- /* Disable registry overrides */}}
+        {{- $ = coll.Omit "kube" $ }}
+
+        {{- /* Set DNS annotations */}}
+        {{- define "svc_annotation" }}
+        services:
+          {{ index . 0 }}:
+            annotations:
+              "external-dns.alpha.kubernetes.io/hostname": {{ index . 1 }}
+        {{- end }}
+        {{- $d := .system_domain }}
+        {{- $ = printf "%s, *.%s" $d $d | slice "router" | tmpl.Exec "svc_annotation" | yaml | merge $ }}
+        {{- $ = printf "ssh.%s" $d | slice "ssh-proxy" | tmpl.Exec "svc_annotation" | yaml | merge $ }}
+        {{- $ = printf "tcp.%s, *.tcp.%s" $d $d | slice "tcp-router" | tmpl.Exec "svc_annotation" | yaml | merge $ }}
+
+        {{- $ | toYAML }}
+        EOF
+      env:
+        BACKEND: gke
+        AUTOSCALER: "true"
+
+    - run: make cf-operator-apply
+      env:
+        CF_OPERATOR_URL: ${{ github.workspace }}/kubecf-bundle/cf-operator.tgz
```

Comment:
Why is this required?

Reply:
Given that we already have it (because we have a bundle), downloading it again seems dumb.  This also ensures we don't accidentally use `latest`.

Reply:
I would expect `make cf-operator-apply` to install kubecf's dependency.
If it doesn't, it's a bug.

Reply:
> I would expect `make cf-operator-apply` to install kubecf's dependency.

It does `helm install cf-operator "$(cf_operator_url)" ...` and `cf_operator_url` will return the resolved URL from `dependencies.yaml`, i.e. `https://s3.amazonaws.com/cf-operators/release/helm-charts/cf-operator-{version}.tgz` with `{version}` replaced by the version also specified in `dependencies.yaml`.